packages/desktop-client/src/components/payees/PayeeTableRow.tsx

```diff
@@ -47,8 +47,12 @@ function RuleButton({ ruleCount, focused, onEdit, onClick }: RuleButtonProps) {
       >
         <Text style={{ paddingRight: 5 }}>
           {ruleCount > 0 ? (
-            <Trans>
-              {ruleCount} associated {ruleCount === 1 ? 'rule' : 'rules'}
+            <Trans
+              i18nKey="associatedRule"
+              count={ruleCount}
+              values={{ ruleCount }}
+            >
+              {{ ruleCount }} associated rule
```

Comment:
It doesn't look like this is pluralising correctly

![image](https://github.com/user-attachments/assets/8cbfc2de-18ec-4517-87d5-5bb3eb16d1ef)

Reply:
I'm sorry.. should've paid attention.

What do you think about

```jsx
<Text style={{ paddingRight: 5 }}>
  {ruleCount > 0 ? (
    <Trans>
      {`${ruleCount} associated rule${ruleCount === 1 ? '' : 's'}`}
    </Trans>
  ) : (
    <Trans>Create rule</Trans>
  )}
</Text>
```

I don't quite like it but I don't know how else do solve it without having a `en.json`.